### PR TITLE
Responsive google maps

### PR DESCRIPTION
**Pull request by nickserv**
Pulled from https://github.com/paircolumbus/paircolumbus.github.io/tree/responsive-google-maps and rebased on master.

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -53,3 +53,17 @@ $on-laptop:        800px;
...
+    padding-top: 30px;
+    overflow: hidden;
+    margin: 0 auto;
+}
+.map-container iframe {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+}
```
Comment (nickserv) [inline css/main.scss:69]: I'm kind of confused with how this CSS works. Is all of this necessary? Where is there a large `padding-bottom` that we're not seeing? I assume that's from the `overflow: hidden;`.

**Comment by nickserv** [comment]
I think this looks pretty good on mobile. That being said, I'm a little concerned with how long this may take to load on mobile devices. Fortunately, the Google Maps widget loads after page load so it won't delay other page content, but it still makes it take longer to see the address. How would you feel about keeping the address, but having it link to the address on Google Maps? This has the advantage that the user can choose if they want to load Google Maps, and if they do on their mobile device it can open in the native app.

I could be thinking about this too much, so feel free to disagree. I do think this would be fine to merge as is though.

**Comment by jaybobo** [comment]
Agreed. Relying on an end-users native app is probably best here. We already have tons of content on index.

**Comment by nickserv** [comment]
@degoeym Thank you for your contribution, but I think I'll close this in favor of replacing the address with a link. I'll probably make a new pull request with that and your wording tweak soon.

**Pull request closed without merging**